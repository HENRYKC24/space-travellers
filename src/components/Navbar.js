import { NavLink } from 'react-router-dom';
import Image from '../logo.png';

const Navbar = () => (
  <nav className="navbar">
    <img src={Image} alt="space x" width="80" height="80" />

    <ul className="nav">
      <li><NavLink to="/" exact>Rockets</NavLink></li>
      <li><NavLink to="/missions" exact>Missions</NavLink></li>
      |
      <li><NavLink to="/profile" exact>My Profile</NavLink></li>
    </ul>
  </nav>
);

export default Navbar;
