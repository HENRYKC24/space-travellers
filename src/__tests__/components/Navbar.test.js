import React from 'react';
import { BrowserRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';
import Navbar from '../../components/Navbar';

it('should render correctly', () => {
  const tree = renderer.create(<BrowserRouter><Navbar /></BrowserRouter>).toJSON();
  expect(tree).toMatchSnapshot();
});
