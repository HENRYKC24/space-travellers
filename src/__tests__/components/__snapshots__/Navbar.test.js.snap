// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly 1`] = `
<nav
  className="navbar"
>
  <div
    className="app-name-logo"
  >
    <a
      aria-current="page"
      className="active"
      href="/"
      onClick={[Function]}
      style={Object {}}
    >
      <img
        alt="space x"
        height="60"
        src="logo.png"
        width="60"
      />
    </a>
    <a
      aria-current="page"
      className="active"
      href="/"
      onClick={[Function]}
      style={
        Object {
          "color": "#000",
          "textDecoration": "none",
        }
      }
    >
      <h1
        className="app-name"
      >
        Space Travellers
      </h1>
    </a>
  </div>
  <ul
    className="nav"
  >
    <li>
      <a
        aria-current="page"
        className="active"
        href="/"
        onClick={[Function]}
        style={Object {}}
      >
        Rockets
      </a>
    </li>
    <li>
      <a
        aria-current={null}
        href="/missions"
        onClick={[Function]}
      >
        Missions
      </a>
    </li>
    <li
      className="pipe"
    >
      |
    </li>
    <li>
      <a
        aria-current={null}
        href="/profile"
        onClick={[Function]}
      >
        My Profile
      </a>
    </li>
  </ul>
</nav>
`;
